Classes/Private/Wrapper/OCTToxAV.m

```diff
@@ -258,7 +388,16 @@ void callstateCallback(ToxAV *cToxAV,
                        uint32_t state,
                        void *userData)
 {
-    // To Do..
+    OCTToxAV *toxAV = (__bridge OCTToxAV *)userData;
+
+    dispatch_async(dispatch_get_main_queue(), ^{
+        DDLogCInfo(@"%@: callstateCallback from friend %d with state: %d", cToxAV, friendNumber, state);
+
+
+        if ([toxAV.delegate respondsToSelector:@selector(toxAV:callStateChanged:friendNumber:)]) {
+            [toxAV.delegate toxAV:toxAV callStateChanged:(OCTToxAVCallState)state friendNumber:friendNumber];
```

Comment:
I am also adding more test cases and it seems to work for now
https://gist.github.com/Chuongv/cbfcc013741ab7c9851d
Let me know if I am overlooking something